src/drive/targets/manifest.webapp

```diff
@@ -180,7 +180,7 @@
   "accept_documents_from_flagship": {
     "accepted_mime_types": ["*/*"],
     "max_number_of_files": 1,
-    "max_size_per_file_in_MB": 10,
+    "max_size_per_file_in_MB": 100,
```

Comment:
No way. We can't handle 100MB.

Reply:
> No way. We can't handle 100MB.

what max value should we use? 10MB seems too low for me as even pictures can be above that

Reply:
We have speced 10MB. No?

Reply:
> We have speced 10MB. No?

I believe that was for mespapiers only, could not find a hard number for Drive

Reply:
Use 10MB too. We'll see later.